### PR TITLE
[Changes made] Update free-courses-en.md

### PR DESCRIPTION
**Pull request by chhavibansal**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by chhavibansal** [comment]
Title: Added Scala tutorials

Description:
Added a collection of scala tutorials to the relevant section in free-courses-en.md.

Purpose:
This addition provides learners with valuable resources to explore scala, enhancing the educational content available in this repository.

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -1610,6 +1610,8 @@
 ### Scala
 
 * [Functional Programming in Scala Specialization](https://www.coursera.org/course/reactive)
+* [Scala & Functional Programming Essentials](https://www.udemy.com/course/rock-the-jvm-scala-for-beginners/)
+* [Scala Tutorial](https://youtu.be/-8V6bMjThNo?si=4QDqJthhYQiAjB02)
```
Comment (Thenlie) [inline courses/free-courses-en.md:1614]: For youtube, we prefer "playlist" links and the name of the producer/creator. View [these docs](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md#platforms-and-access-notes) for more information. Once you have updated you PR, you can add “changes made” to the title and reviewer will take another look as soon as they are available. Thanks!

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -1610,6 +1610,8 @@
 ### Scala
 
 * [Functional Programming in Scala Specialization](https://www.coursera.org/course/reactive)
+* [Scala & Functional Programming Essentials](https://www.udemy.com/course/rock-the-jvm-scala-for-beginners/)
```
Comment (Thenlie) [inline courses/free-courses-en.md:1613]: This course does not appear to be free when I look at it.

**Review by Thenlie — CHANGES_REQUESTED** [review]

File: courses/free-courses-en.md (hunk 2 above)
Comment (chhavibansal) [reply courses/free-courses-en.md]: @Thenlie updated the PR please can you review

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -1610,6 +1610,7 @@
 ### Scala
 
 * [Functional Programming in Scala Specialization](https://www.coursera.org/course/reactive)
+* [Scala Tutorial](https://www.youtube.com/playlist?list=PLmtsMNDRU0BxryRX4wiwrTZ661xcp6VPM) - Rock the JVM (Youtube)
```
Comment (eshellman) [inline courses/free-courses-en.md:1613]:
```suggestion
* [Scala at Light Speed](https://www.youtube.com/playlist?list=PLmtsMNDRU0BxryRX4wiwrTZ661xcp6VPM) - Rock the JVM
```
Don't invent titles! Use the title given by the resource.

Comment (chhavibansal) [reply courses/free-courses-en.md]: @eshellman thanks for pointing it out. updated. can u please re-review

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**